src/app/dashboard/dashboard-editor/dashboard-editor.component.ts

```diff
@@ -16,12 +16,10 @@ const selector = 'dashboard-editor';
     viewProviders: [DragulaService]
 })
 export class DashboardEditorComponent {
-    private dashboardService;
-    private router;
     private widgets;
     private initialData;
 
-    constructor(dashboardService: DashboardService, router: Router) {
+    constructor(private dashboardService: DashboardService, private router: Router) {
```

Comment:
Actually you can omit assignment in constructor when you mark parameters with access modifier